sdk/identity/identity/src/credentials/interactiveBrowserCredential.ts

```diff
@@ -15,46 +14,30 @@ const SERVER_PORT = process.env.PORT || 80;
 import express from "express";
 import { PublicClientApplication, TokenCache, AuthorizationCodeRequest } from "@azure/msal-node";
 import open from "open";
-import path from "path";
 import http from "http";
 
-const BrowserNotSupportedError = new Error(
-  "InteractiveBrowserCredential is not supported in Node.js."
-);
 const logger = credentialLogger("InteractiveBrowserCredential");
 
-interface AuthenticationRecord {
-  authority?: string;
-  homeAccountId: string;
-  environment: string;
-  tenantId: string;
-  username: string;
-}
-
 /**
  * Enables authentication to Azure Active Directory inside of the web browser
  * using the interactive login flow, either via browser redirects or a popup
  * window.  This credential is not currently supported in Node.js.
  */
 export class InteractiveBrowserCredential implements TokenCredential {
-  private identityClient: IdentityClient;
   private pca: PublicClientApplication;
   private msalCacheManager: TokenCache;
   private tenantId: string;
   private clientId: string;
   private persistenceEnabled: boolean;
   private redirectUri: string;
   private authorityHost: string;
-  private authenticationRecord: AuthenticationRecord | undefined;
```

Comment:
@jonathandturner This was added in #10994
Was curious as to how you envisioned this being used as it is currently unused

Reply:
running `rushx lint` command pointed out that `authenticationRecord` is not in use

Reply:
@mohsin-mehmood Looks like the latest changes in the master branch do make use of the `authenticationRecord`. Can you pull in the changes from master and update the PR?